qemu/tests/rng_stress.py

```diff
@@ -38,9 +38,10 @@ def get_available_rng(session):
         Get available rng devices from /sys/devices
         """
         verify_cmd = params["driver_available_cmd"]
+        driver_name = params["driver_name"]
         try:
             output = session.cmd_output_safe(verify_cmd)
-            rng_devices = re.findall(r"virtio_rng.\d+", output)
+            rng_devices = re.findall(r"(%s(?:\.\d+)?)" % driver_name, output)
```

Comment:
@ybduan  Why use two parentheses in this regular expression,  `re.findall(r"%s(?:\.\d+)?" % driver_name, output)` is enough ?

Reply:
Yes, you're right. Updated, thanks!